gigascience/cms
# Team Members (#4)

# Pull request for issue: #4

This is a pull request for the following functionalities:

* Team member section -- CMS editing interface

## How to test?

Project config must be applied in the container; to do that, run the command
```
docker-compose run --rm php craft project-config/apply
```

Reviewers can add and edit Team Members (except see Issues below)

1. Go to the CMS dashboard at /admin
2. Log in to the CMS with admin credentials (these will be set by the GigaScience devops folks)
3. Click the **Entries** menu item on the left-hand nav
4. Click **Team Members** to open the Team Members section
5. Click **New Entry** at top right (the correct section will be selected automatically)
6. Add and edit content accordingly.

## How have functionalities been implemented?

This is all implemented through Craft CMS; the fields and sections are created through the admin interface by using the **Settings** menu and various submenus. Using the Content Audit document as the source of truth, I did the following:
1. Created a field group using Settings -> Fields
2. Created the fields within the group
3. Created a Channel-type section using Settings -> Sections
4. Created the field layout (this is done via the Entry Type for the Section) 

## Issues with your implementation

The Wasabi bucket does not currently seem to be working as an S3 alternative; more investigation is ongoing. Testing *can be completed* except for adding images. 

## IMPORTANT NOTE:

As these pull requests build on each other (I have to base each branch on the previous branch), this PR also includes the commits from the previous PR. Resolve the PRs in date order!
